src/task/create-item-validation-task.ts

```diff
@@ -0,0 +1,142 @@
+// global
+import { Actor, DatabaseTransactionHandler, Item, ItemService, Member, TaskRunner } from 'graasp';
+// local
+import { ItemValidationService } from '../db-service';
+import { BaseValidationTask } from './base-validation-task';
+import {
+  ItemValidationProcesses,
+  ItemValidationReviewStatuses,
+  ItemValidationStatuses,
+  ITEM_TYPE,
+} from '../constants';
+import { getStatusIdByName } from '../utils';
+import { handleProcesses } from '../processes/handler';
+import { FileTaskManager } from 'graasp-plugin-file';
+import { ProcessNotFoundError } from '../errors';
+import { ItemValidationProcess } from '../types';
+
+type InputType = { itemId: string };
+
+export class CreateItemValidationTask extends BaseValidationTask<string> {
+  input: InputType;
+  itemService: ItemService;
+  fTM: FileTaskManager;
+  runner: TaskRunner<Actor>;
+  serviceItemType: string;
+
+  get name(): string {
+    return CreateItemValidationTask.name;
+  }
+
+  constructor(
+    member: Member,
+    validationService: ItemValidationService,
+    itemService: ItemService,
+    fTM: FileTaskManager,
+    runner: TaskRunner<Actor>,
+    serviceItemType: string,
+    input: InputType,
+  ) {
+    super(member, validationService);
+    this.itemService = itemService;
+    this.input = input;
+    this.fTM = fTM;
+    this.runner = runner;
+    this.serviceItemType = serviceItemType;
+  }
+
+  async run(handler: DatabaseTransactionHandler): Promise<void> {
+    this.status = 'RUNNING';
+
+    const { itemId } = this.input;
+
+    // create record in item-validation
+    const iVId = await this.validationService.createItemValidation(
+      itemId,
+      handler,
+    );
+    // variable to indicate if manual review is needed
+    let needReview = false;
```

Comment:
This variable is a bit far from the scope where it is used, it's a good practice to have variables near where they are used.